Makefile

```diff
@@ -44,16 +43,16 @@ clean:
 	rm -rf lib test/test
 
 lint:
-	$(DEPS_BIN_DIR)/eslint src
+	npm run lint
```

Comment:
Quite ironic that we're using npm here actually. Why not use `./lib/cmd.js run lint`?